cmake/std/atdm/tlcc2/all_supported_builds.sh

```diff
@@ -1,5 +1,5 @@
 
-export ATDM_CONFIG_CTEST_S_BUILD_NAME_PREFIX=Trilinos-atdm-chama-
+export ATDM_CONFIG_CTEST_S_BUILD_NAME_PREFIX=Trilinos-atdm-tlcc2-
```

Comment:
@fryeguy52, see this assumes the name of the build scripts will have 'tlcc2' in the name but currently  thoses scripts don't actually exist.